tests/llmobs/test_llmobs_service.py

```diff
@@ -99,10 +101,11 @@ def test_service_enable_no_api_key():
     with override_global_config(dict(_dd_api_key="", _llmobs_ml_app="<ml-app-name>")):
         dummy_tracer = DummyTracer()
         with pytest.raises(ValueError):
-            llmobs_service.enable(_tracer=dummy_tracer)
+            llmobs_service.enable(_tracer=dummy_tracer, agentless_enabled=True)
```

Comment:
With #9912, this will likely be fine to be reverted.

Reply:
We still need this one - or we need config changes passed in to enable agentless.